README.md

```diff
@@ -109,7 +99,10 @@ There's a complete "getting started" guide and more detailed topics available at
 
 Please feel free to [contribute documentation](https://github.com/sbt/sbt-native-packager/tree/master/src/sphinx), or raise issues where you feel it may be lacking.
 
-## Related SBT Plugins
+## Related SBT Plugins ##
+
+These are a list of plugins that either use sbt-native-packager, provide additional features
+or provide a richer API for a single packaging format.
 
 - [sbt-heroku](https://github.com/heroku/sbt-heroku)
```

Comment:
While the README is being updated, how about putting this list in alphabetical order?